### ISSUE TITLE
[bug] TrueAnal matches not working

### ISSUE THREAD

**Issue by PAHelper2** [PAhelper/PhoenixAdult.bundle#18]
When trying to match using any of these:

TrueAnal - Back in Gia's Ass
TrueAnal - Back in Gias Ass
True Anal - Back in Gia's Ass
True Anal - Back in Gias Ass

No results are ever found.  Trying to use the search function on trueanal.com's site also fails, however, to find anything.  There are only search results found if you let the site do auto-search while typing in the search bar.  Putting in a search term, hitting return, never seems to find anything via:

https://tour.trueanal.com/search/back%20in%20gia's%20ass

So it seems like the search url used originally just isn't functional anymore.

**Comment by PAHelper2** [comment]
The search preview in the search bar will also fail if you get to the apostrophe in that string.  So it doesn't appear to be encoding properly.

**Comment by MrDemous** [comment]
Try Trueanal - Back and you'll get a list of results with Back in the title with Back in Gia's Ass at the top.

**Comment by PAHelper2** [comment]
I can confirm that @MrDemous suggestion works.  It even works if I go all the way to "Trueanal - Back in Gia".  So basically it errors because trueanal.com doesn't properly handle the apostrophe.

**Comment by PAhelper** [comment]
I tried several variations of this search to try and find a programmatic way to resolve it, and the only one I came up with might cause other problems. As you noted, if you omit the apostrophe it fails, if you replace it with a space it fails, if you omit the entire word (ie. "Back in Ass") it fails. The only way I could possibly think to solve this is to strip all characters including and past the apostrophe, but what about a title like "Lilly's first time"? Your search title would end up just being "Lilly". I'm afraid this solution might cause more problems that good.
